connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTaskContext.java

```diff
@@ -109,4 +111,13 @@ public void resume(TopicPartition... partitions) {
     public Set<TopicPartition> pausedPartitions() {
         return pausedPartitions;
     }
+
+    @Override
+    public void disableOffsetCommit() {
```

Comment:
There seems not. The motivation of this patch is to find a mechanism to allow the sink connector to disable offset commit in the framework. I have also thought about other mechanism such as adding a common connector config in 'ConnectorConfig``. The problem with the config approach is that it is a bit hard to only allow sink connectors to use that.  Thoughts?